.github/workflows/script_lint.yml

```
@@ -20,8 +20,21 @@ jobs:
           python-version: "3.10"
       - name: flake8 Lint
         uses: py-actions/flake8@v2
-        with:
-           path: common/CI common/Hooks common/Scripts/worklog.py
+        with: 
+           path: >_
+              common/CI
+              common/Hooks
+              common/Scripts/worklog.py
+              common/Scripts/dloader.py
+              common/Scripts/epcsearch.py
+              common/Scripts/ep-update.py
+              common/Scripts/find_deps.py
+              common/Scripts/gettag.py
+              common/Scripts/latent-rebuilds.py
+              common/Scripts/pbump.py
+              common/Scripts/worklog.py
+              common/Scripts/yauto.py
+              common/Scripts/yconvert.py 
```

Should be `>-` and you can add `common/Scripts` to ensure all scripts are checked:

```suggestion
           path: >-
              common/CI
              common/Hooks
              common/Scripts
```